CatNote.API/Program.cs

```diff
@@ -1,14 +1,18 @@
-using CatNote.API.Mappers;
+﻿using CatNote.API.Mappers;
 using CatNote.API.Middlewares;
 using CatNote.BLL.DI;
-using FluentValidation.AspNetCore;
-using System.Reflection;
 using CatNote.BLL.Mappers;
+using FluentValidation.AspNetCore;
 
 namespace CatNote.API;
 
 public class Program
 {
+    private Program()
+    {
+        
```

Comment:
What empty private constructor is doing here?

Reply:
Sonar gave a warning when I added static or protected I got an exception
![image](https://github.com/Finka95/CatNote/assets/76497091/bfe77d8e-a1cb-4852-8ad2-9ed1660e6914)